README.md

```diff
@@ -66,7 +66,7 @@ OPTIONS
   --language=language    ISO language code or "NONE" to not specify a language. Defaults to the OS locale
 ```
 
-_See code: [src/commands/edge/channels.ts](https://github.com/SmartThingsCommunity/edge-cli-plugin/blob/v0.0.0/src/commands/edge/channels.ts)_
+_See code: [dist/commands/edge/channels.ts](https://github.com/SmartThingsCommunity/edge-cli-plugin/blob/v1.0.4/dist/commands/edge/channels.ts)_
```

Comment:
If these are using `dist`, the links will 404.

Reply:
Well, shoot. I'll look into seeing if there is any way to configure oclif-dev. It seems to come from this line in package.json:

```
	"oclif": {
		"commands": "./dist/commands",
```

but if we use `src` there, the packaged (with pkg) binary throws an error when you run an edge command.

Reply:
Ok, I think I know how to fix this now. Running `oclif-dev readme --help` says:

> Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.

I'll open an issue for this.